pages/profiles/graph.svg.js

```diff
@@ -0,0 +1,29 @@
+// node_modules
+import Image from "next/image";
+
+// components
+import Breadcrumbs from "../../components/breadcrumbs";
+import PagePreamble from "../../components/page-preamble";
+import { DataPanel, DataAreaTitle } from "../../components/data-area";
+
+export default function GraphSvg() {
+  return (
+    <>
+      <Breadcrumbs />
+      <PagePreamble />
+      <>
```

Comment:
You don’t need the fragments here. Just make `<DataAreaTitle>` and `<DataPanel>` peers of `<Breadcrumbs>` and `<PagePreamble>`.